### PR TITLE
Fix audio only sources

### PR DESCRIPTION
**Pull request by justjanne**
- Alsa should be included in the audio only source list
- Don't try to mirror video for audio only sources
- VoctoGUI relies on VideoDisplay to update the audio levels, create an audio only alternative

### CODE REVIEW

File: voctogui/lib/audioonlydisplay.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        self.pipeline.set_state(Gst.State.PLAYING)
+
+    def on_error(self, bus, message):
+        (error, debug) = message.parse_error()
+        self.log.error(
+            "GStreamer pipeline element '%s' signaled an error #%u: %s" % (message.src.name, error.code, error.message))
+
+    def mute(self, mute):
+        self.pipeline.get_by_name("audiosink-{name}".format(name=self.name)).set_property(
+            "volume", 1 if mute else 0)
```
Comment (Kunsi) [inline voctogui/lib/audioonlydisplay.py:90]: Great, this is inverted (but also inverted in `videodisplay.py`, so let's keep this here as well.

**Review by Kunsi — APPROVED** [review]
Thanks! Do you have a screenshot of how this looks in voctogui?

**Comment by justjanne** [comment]
> Thanks! Do you have a screenshot of how this looks in voctogui?

The audioonlysource doesn't show up at all in the video sources on the left, but it does show up in the audio tab on the bottom. It looks just like you'd expect.

I'll try to set up the encoder in the next few days again, then I'll be able to attach a screenshot as well.

File: voctogui/lib/audioonlydisplay.py (hunk 1 above)
Comment (justjanne) [reply voctogui/lib/audioonlydisplay.py:90]: Tbh, this is just a quick workaround - a better long-term solution would probably be for videodisplay to handle optional audio or optional video itself.

**Comment by Kunsi** [comment]
Atleast for this configuration, `livesources=slides` is audio-only:

```ini
[mix]
videocaps=video/x-raw,format=I420,width=1920,height=1080,framerate=25/1,pixel-aspect-ratio=1/1,interlace-mode=progressive,colorimetry=bt709
audiocaps=audio/x-raw,format=S16LE,channels=8,layout=interleaved,rate=48000

sources=cam1,cam2,cam3,slides
livesources=slides

[source.cam1]
kind=decklink
devicenumber=1
video_connection=SDI
video_mode=1080p25
volume=1.0
audio.original = 0+1

[source.cam2]
kind=test
volume=1.0
audio.test = 0+1

[source.cam3]
kind=test
volume=0.0

[source.slides]
kind=decklink
devicenumber=2
video_connection=SDI
video_mode=1080p25
volume=0.0
```

Will try to find out why, if i can't figure it out i might have to revert this.

**Comment by Kunsi** [comment]
fixed in 1563f9ec90ee0e21e70a0fd8031fe6c0b81b420e - no clue why noone spotted this earlier

**Pull request merged**